frontend/src/app/pages/exercises/exercise/shared/emergency-operations-center/send-alarm-group-interface/send-alarm-group-interface.component.ts

```diff
@@ -85,10 +94,54 @@ export class SendAlarmGroupInterfaceComponent implements OnDestroy {
         );
         this.firstVehiclesCount -= firstVehiclesCountReducedBy;
 
+        const vehicleTemplates = selectStateSnapshot(
+            selectVehicleTemplates,
+            this.store
+        );
+        const vehicleTemplatesById = Object.fromEntries(
+            vehicleTemplates.map((template) => [template.id, template])
+        );
+
+        const materialTemplates = selectStateSnapshot(
+            selectMaterialTemplates,
+            this.store
+        );
+        const personnelTemplates = selectStateSnapshot(
+            selectPersonnelTemplates,
+            this.store
+        );
+
+        const sortedAlarmGroupVehicles = StrictObject.values(
+            alarmGroup.alarmGroupVehicles
+        ).sort((a, b) => a.time - b.time);
+
+        // We have to provide a map position when creating a vehicle
+        // It will be overwritten directly after by putting the vehicle into transfer
+        const placeholderPosition = MapCoordinates.create(0, 0);
+
+        // Create vehicles for the alarm group
```

Comment:
This is not quite correct, right? We're not creating vehicles but rather vehicle parameters.

Reply:
Nice catch. Corrected in c6d6d1acced92b6ca4b76bbc6266c347e0cbd592